task/01-strings-tasks.js

```diff
@@ -236,9 +261,13 @@ function encodeToRot13(str) {
  *   isString({}) => false
  *   isString('test') => true
  *   isString(new String('test')) => true
- */
+ */ 
 function isString(value) {
```

Comment:
check preconditions first, avoid unnecessary branch nesting, do not modify local variables if possible